core/src/test/java/org/apache/calcite/util/graph/DirectedGraphTest.java

```diff
@@ -153,8 +153,12 @@
    * {@link DefaultDirectedGraph#removeAllVertices(java.util.Collection)}. */
   @Test void testRemoveAllVertices() {
     final DefaultDirectedGraph<String, DefaultEdge> graph = createDag();
+    assertEquals(6, graph.edgeSet().size());
     graph.removeAllVertices(Arrays.asList("B", "E"));
     assertEquals("[A, C, D, F]", graph.vertexSet().toString());
+    // you can find more complete case in HepPlannerTest.testCleanEdges
```

Comment:
Sorry. One more comment: please remove this comment

Reply:
thanks for your reminder, I have removed it now.